### ISSUE TITLE
`tabIndex={0}` does not work for accessibility of iOS

### ISSUE THREAD

**Issue by imWildCat** [tamagui/tamagui#2865]
### Current Behavior

accessible is deprecated in tamagui

but `tabIndex={0}` does not work

### Tamagui Version

```markdown
1.108.3 or 4
```

### Platform (Web, iOS, Android)

```markdown
iOS at least
```

### Reproduction

```markdown
https://github.com/imWildCat-archived/tamagui-tabindex-does-not-work/commit/d2883b23fdf052a4d7df49a73b3892d4ec69d401
```

**Comment by hackily** [comment]
+1 I've been having to provide the `accessible` prop despite it being deprecated

**Comment by thienautran** [comment]
+2 same issue. I've needed to give accessible despite it being deprecated

**Comment by imWildCat** [comment]
I'm closing this issue because I've migrated my app out of Tamagui into http://nativewind.dev instead.

@hackily @thienautran please feel free to create a new one.
